game-core/src/main/java/tools/image/PolygonGrabber.java

```diff
@@ -678,9 +678,8 @@ private Polygon findPolygon(final int x, final int y) {
                 + "and removing any anti-aliasing.");
         return null;
       }
-      int tempDirection;
       for (int i = 2; i >= -3; i--) { // was -4
-        tempDirection = (currentDirection + i) % 8;
+        int tempDirection = (currentDirection + i) % 8;
         if (tempDirection < 0) {
           tempDirection += 8;
         }
```

Comment:
```java
final int tempDirection = Math.floorMod(currentDirection, 8);
```

Reply:
`currentDirection + i` of course

Reply:
1a643cb

While there I also cleaned up the javadocs and deprecated the class as it's one-time only per commentary.